### ISSUE TITLE
Why is the size of the checkpoint saved during training large?

### ISSUE THREAD

**Issue by haubui17062019** [SWivid/F5-TTS#352]
I fine tune the F5-TTS model, however I see that the weight size of the model is more than 5GB while your weight size is only 1.26GB. What is the reason?

**Comment by ZhikangNiu** [comment]
because we re-clean the train checkpoint and only save ema_model_state_dict

**Comment by SWivid** [comment]
during training, the ckpt is saving also the states of optimizer, scheduler, model, ema_model, etc.

will close this issue, feel free to open if further questions~
